bin/picca_compute_fvoigt_hcd.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
+import argparse
+
+import numpy as np
+from scipy.constants import speed_of_light
+from scipy.special import wofz
+
+from picca import constants
+
+
+def voigt_profile(x, sigma=1, gamma=1):
+    return np.real(wofz((x + 1j*gamma) / (sigma * np.sqrt(2))))
```

Comment:
is there a reason to define the voigt profile function here instead of just using `scipi.special.voigt_profile` in the first place?

Reply:
I think the difference might be in using the Voigt function V in case of scipy and H in case of this computation, defined like this on wikipedia 
![image](https://github.com/igmhub/picca/assets/3012942/2aa32b04-a95d-40d1-a171-f74c0b5af2ac).
Anyway, using wofz or voigt_profile straight from scipy is fine (with the correct normalizations), just improves readability as you don't need to redefine things here, scipy is using wofz internally anyway (but in a c-based routine).

Reply:
If both are the same (once properly normalized), I would suggest changing this to using `scipi.special.voigt_profile` given that this is what we use in the DLA masking module, but this would not be a show stopper for the merge

Reply:
they are as the scipy.special.voigt_profile is just a cythonized wrapper around woffz anyway, I'd mildly prefer using existing functions (with some comment about the difference in intrinsic normalization) over having our own version for smaller things like this, even more given we're using the scipy function elsewhere anyway...